web/src/components/organisms/Dashboard/index.tsx

```diff
@@ -1,27 +1,32 @@
-import styled from "@emotion/styled";
-import { useState } from "react";
+import { useCallback, useState } from "react";
 import { useParams } from "react-router-dom";
 
-import Content from "@reearth-cms/components/atoms/Content";
-import Header from "@reearth-cms/components/atoms/Header";
-import Layout from "@reearth-cms/components/atoms/Layout";
-import Sider from "@reearth-cms/components/atoms/Sider";
 import MoleculeHeader from "@reearth-cms/components/molecules/Common/Header";
-import ProjectMenu from "@reearth-cms/components/molecules/Common/ProjectMenu";
 import WorkspaceCreationModal from "@reearth-cms/components/molecules/Common/WorkspaceCreationModal";
-import WorkspaceMenu from "@reearth-cms/components/molecules/Common/WorkspaceMenu";
+import DashboardMolecule from "@reearth-cms/components/molecules/Dashboard";
 
 import useHooks from "./hooks";
 
-type Props = {
-  children?: React.ReactNode;
+type InnerProps = {
+  onWorkspaceModalOpen?: () => void;
+};
+
+export type Props = {
+  child: React.FC<InnerProps>;
   defaultSelectedKeys?: string[];
-  menuType?: "project" | "workspace";
+  sidebar: React.FC<{
+    projectId?: string;
+    inlineCollapsed: boolean;
+    isPersonalWorkspace?: boolean;
+    workspaceId?: string;
+    defaultSelectedKeys?: string[];
+  }>;
 };
 
-const Dashboard: React.FC<Props> = ({ children, defaultSelectedKeys, menuType }) => {
+const Dashboard: React.FC<Props> = ({ child: Child, defaultSelectedKeys, sidebar: Sidebar }) => {
```

Comment:
This isn't so much a dashboard anymore as much as it is a page wrapper. Do you have any better ideas for naming or is PageWrapper reasonable?

Reply:
@KaWaite , I guess `CMSWrapper` is a good name, all CMS pages are used inside this component.